drools-eclipse/pom.xml

```diff
@@ -117,7 +117,7 @@
           <artifact>
             <groupId>org.jboss.tools.integration-stack</groupId>
             <artifactId>target-platform</artifactId>
-            <version>4.3.0.Beta1f-SNAPSHOT</version>
+            <version>4.3.1-SNAPSHOT</version>
```

Comment:
This is the version that Paul suggested I use...for now. I'm sure this will
change in the near future :-/

On Tue, Nov 24, 2015 at 6:51 PM, Kris <notifications@github.com> wrote:

> In drools-eclipse/pom.xml
> <https://github.com/droolsjbpm/droolsjbpm-tools/pull/37#discussion_r45821434>
> :
>
> > @@ -117,7 +117,7 @@
> >            <artifact>
> >              <groupId>org.jboss.tools.integration-stack</groupId>
> >              <artifactId>target-platform</artifactId>
> > -            <version>4.3.0.Beta1f-SNAPSHOT</version>
> > +            <version>4.3.1-SNAPSHOT</version>
>
> Bob, is there a need to use a SNAPSHOT here, what about staying on the
> latest stable release instead? While in theory we can live with SNAPSHOTS
> in between releases, I'm a little afraid we'd forget about it and only
> discover potential issues too late when we try to do a release.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/droolsjbpm/droolsjbpm-tools/pull/37/files#r45821434>.
>



-- 
________________________
Robert ("Bob") Brodt
Senior Software Engineer
JBoss by Red Hat